### ISSUE TITLE
the model.pth file is too large

### ISSUE THREAD

**Issue by dreamhighchina** [tianzhi0549/FCOS#191]
when i have trained fcos_R_101_FPN_2x,the model.pth is 399M which is  huge different from your 200M.my class is 7.could tell me the possible reason???

**Comment by milliema** [comment]
Same for me. The pretrained model provided by the author is 133M, but after fine tune on my own dataset, the model size increased to 260M, doubled. I guess it may be caused by the format of numbers, e.g. changing from float16 to float32 after training?

**Comment by milliema** [comment]
I found the real reason, after removing the scheduler, optimizer, and iteration, the model size will be decreased to the same as author provided.

**Comment by dreamhighchina** [comment]
how to remove these？

**Comment by milliema** [comment]
in tools/remove_solver_states.py

**Comment by dreamhighchina** [comment]
> 
> 
> in tools/remove_solver_states.py

thank you for your reply，how can i communicate with you to talk about this model about other questions？
